presto-native-execution/presto_cpp/main/operators/BroadcastFactory.cpp

```diff
@@ -115,6 +127,7 @@ void BroadcastFileWriter::write(const RowVectorPtr& rowVector) {
 
   initializeWriteFile();
   numRows_ += numRows;
+  retainedSize_ += rowVector->retainedSize();
```

Comment:
RetainedSize depends on vector encoding. I think we should use estimateFlatSize instead.

This commits add both size and number of rows. Please, update commit message to reflect that. Also, make sure that all lines are within 78 chars in the commit message.

Reply:
As discussed offline, updated code with maxSerializedSize.